opencog/atoms/execution/ExecutionOutputLink.h

```diff
@@ -63,6 +64,15 @@ static inline ExecutionOutputLinkPtr ExecutionOutputLinkCast(AtomPtr a)
 // XXX temporary hack ...
 #define createExecutionOutputLink std::make_shared<ExecutionOutputLink>
 
+class LibraryManager
+{
+private:
+    static std::unordered_map<std::string, void*> _librarys;
+    static std::unordered_map<std::string, void*> _functions;
+public:
+    static void* getFunc(std::string libName,std::string funcName);
+};
+
```

Comment:
could you move this class declaration out of the header file, and into the C file? I see no reason to make this public --  its not even a real class, as all of it's members are static. Its also specific to linux; a mac or windows port would work differently.